scripts/eagle_renumber.py

```diff
@@ -78,14 +78,14 @@ def resolve_unit(value, unit_prefix):
       '<part name="{}(.+?)".*value="([\.0-9]+)([{}]?)(.*?)".*[/]?>'.\
             format(prefix, ''.join(known_unit_prefixes.keys()))
             )
-sch_instance_tag_re = re.compile('<instance part="{}(.+?)".*'.format(prefix))
-sch_pinref_tag_re = re.compile('<pinref part="{}(.+?)".*'.format(prefix))
+sch_instance_tag_re = re.compile('<instance part="{}([0-9]+?)".*'.format(prefix))
```

Comment:
Wouldn't that only be valid in the value? Or do we want to allow periods in the part names too?

Either way I don't think there is a problem with making the matching more inclusive.